lib/rspec/active_job/enqueue_a.rb

```diff
@@ -101,6 +119,14 @@ def new_jobs_with_correct_class_and_args
         def enqueued_jobs
           ::ActiveJob::Base.queue_adapter.enqueued_jobs
         end
+
+        def at_correct_time?
+          return true unless run_time
+
+          job = new_jobs_with_correct_class.first
```

Comment:
I think this should probably do a `find` instead of fixing on the first - I don't think the test should fail if you've enqueued one to run in 5 minutes and one to run in 10, and do `to_run_at(10.minutes.from_now)`